week-1/I-floats/exercise.js

```diff
@@ -1,2 +1,13 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+var total = numberOfMentors + numberOfStudents;
+var studentPercentage = numberOfStudents / total;
+var studentRoughPercentage = Math.round(100 * studentPercentage);
+var mentorPercentage = numberOfMentors / total;
+var mentorRoughPercentage = Math.round(100 * mentorPercentage);
+
+var message1 = "Percentage students : " + studentRoughPercentage + "%";
```

Comment:
good job

Reply:
Thanks Mussie.

Reply:
nice job I had to comment as well, hopefully next time I will give a useful one lol

Reply:
Abz, thanks for you kind words.

Reply:
easy to understand good work :)

Reply:
Thank you very much, Nina.